README.md

```diff
@@ -52,8 +52,8 @@ ensure org-mode buffers are not dimmed.
 * `dimmer-configure-posframe` is a convenience function for posframe
 users to ensure posframe buffers are not dimmed.
 
-* `dimmer-configure-which-key` is a convenience function for which-key
-users to ensure which-key popups are not dimmed.
+* `dimmer-configure-magitis a convenience function for magit users to
+ensure transients are not dimmed.
```

Comment:
typo: you're missing the closing quote and a space is needed after `dimmer-configure-magit`